// firebaseConfig/firebaseConfig.js
import firebase from "firebase/app";
import "firebase/auth";
import "firebase/database";

const firebaseConfig = {
  apiKey: "AIzaSyBvK8CuhugIrqT6linCkWKcb_Hvby_6WJc",
  authDomain: "blogsite-c1c0a.firebaseapp.com",
  databaseURL: "https://blogsite-c1c0a-default-rtdb.firebaseio.com",
  projectId: "blogsite-c1c0a",
  storageBucket: "blogsite-c1c0a.appspot.com",
  messagingSenderId: "228529823576",
  appId: "1:228529823576:web:f02492093b97c62c4fcf2d",
  measurementId: "G-JNKMP0HSVX"
};



firebase.initializeApp(firebaseConfig);
const projectDB = firebase.database();
const auth = firebase.auth();
// Import the functions you need from the SDKs you need
// import { initializeApp } from "firebase/app";
// import { getAnalytics } from "firebase/analytics";
// TODO: Add SDKs for Firebase products that you want to use
// https://firebase.google.com/docs/web/setup#available-libraries

// Your web app's Firebase configuration
// For Firebase JS SDK v7.20.0 and later, measurementId is optional


// Initialize Firebase
// const app = initializeApp(firebaseConfig);
// const analytics = getAnalytics(app);

export { projectDB, auth };
